### PR TITLE
fix structure logging format error

### PR DESCRIPTION
**Pull request by lala123912**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
structure logging format error, not meet the requirement of  kubernetes/enhancements#1602
#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

@Sharma
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kubelet/kuberuntime/kuberuntime_manager.go [hunk 1]
```diff
@@ -974,7 +974,7 @@ func (m *kubeGenericRuntimeManager) GetPodStatus(uid kubetypes.UID, name, namesp
 
 	podFullName := format.Pod(pod)
 
-	klog.V(4).InfoS("getSandboxIDByPodUID got sandbox IDs for pod", "podSandboxID", podSandboxIDs, "pod", klog.KObj(pod))
+	klog.V(4).InfoS("GetSandboxIDByPodUID got sandbox IDs for pod", "podSandboxID", podSandboxIDs, "pod", klog.KObj(pod))
```
Comment (navidshaikh) [inline pkg/kubelet/kuberuntime/kuberuntime_manager.go:977]:
```suggestion
	klog.V(4).InfoS("getSandboxIDByPodUID got sandbox IDs for pod", "podSandboxID", podSandboxIDs, "pod", klog.KObj(pod))
```

Comment (navidshaikh) [reply pkg/kubelet/kuberuntime/kuberuntime_manager.go:977]: I think we should not change `getSandboxIDByPodUID` as it's a method name reference (see line 962 above)

Comment (lala123912) [reply pkg/kubelet/kuberuntime/kuberuntime_manager.go]: Thanks for reviewing. I have revert the change.

File: pkg/kubelet/kuberuntime/kuberuntime_manager.go [hunk 2]
```diff
@@ -683,7 +683,7 @@ func (m *kubeGenericRuntimeManager) computePodActions(pod *v1.Pod, podStatus *ku
 func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontainer.PodStatus, pullSecrets []v1.Secret, backOff *flowcontrol.Backoff) (result kubecontainer.PodSyncResult) {
 	// Step 1: Compute sandbox and container changes.
 	podContainerChanges := m.computePodActions(pod, podStatus)
-	klog.V(3).InfoS("computePodActions got for pod", "podActions", podContainerChanges, "pod", klog.KObj(pod))
+	klog.V(3).InfoS("ComputePodActions got for pod", "podActions", podContainerChanges, "pod", klog.KObj(pod))
```
Comment (yuzhiquan) [inline pkg/kubelet/kuberuntime/kuberuntime_manager.go:686]: same with [this](https://github.com/kubernetes/kubernetes/pull/100066#discussion_r591671060), `computePodActions` is method name.

Comment (lala123912) [reply pkg/kubelet/kuberuntime/kuberuntime_manager.go]: thank you for review. The code has updated.

**Comment by ehashman** [comment]
/close

this is a duplicate of #99999 but misses one line

**Comment by ehashman** [comment]
Thanks for taking a look at this!

**Pull request closed without merging**